tasks/04_create_links_to_borg_and_borgmatic.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Create links to the Borgmatic and Borg binarys
+  tags:
+    - install_backup
+  block:
+    - name: Create borgmatic command in /usr/local/bin
```

Comment:
This will break things, when packages aren't installed via pip. Just re-arranging this.